services/emailService.js

```diff
@@ -0,0 +1,17 @@
+import sgMail from '@sendgrid/mail';
+import dotenv from 'dotenv';
+
+dotenv.config();
+sgMail.setApiKey(process.env.EMAIL_API_KEY);
+
+
+export default {
+  sendWelcomeMail({ email, subject = 'Notification from LearnGround', html }) {
```

Comment:
The function should be `sendEmail`. The name signifies that it is a generic function that is not particular to `welcome` alone.